js/tabs.js

```diff
@@ -352,20 +348,27 @@ Tabs.prototype.promptSave_ = function(tab, callbackShowDialog) {
   this.dialogController_.show(callbackShowDialog);
 };
 
-Tabs.prototype.save = function(opt_tab, opt_close) {
+/**
+ * Save opt_tab, or the current tab if no opt_tab is passed.
+ * @param {Tab=} opt_tab
+ * @param {function()=} opt_callback
+ */
+Tabs.prototype.save = function(opt_tab, opt_callback) {
   if (!opt_tab)
```

Comment:
I'd suggest aliasing opt_tab to something else. So

const tab = opt_tab || this.currentTab_;

Reply:
Good call. Done and done for saveAs too. Used var instead of const as per es6 problem above.